flutter/cpp/utils.cc

```diff
@@ -105,6 +106,11 @@ mlperf_backend_configuration_t CppToCSettings(const SettingList &settings) {
   strcpy(accelerator, settings.benchmark_setting().accelerator().c_str());
   c_settings.accelerator = accelerator;
   c_settings.batch_size = settings.benchmark_setting().batch_size();
+  char *accelerator_desc =
+      new char[settings.benchmark_setting().accelerator_desc().length() + 1];
+  strcpy(accelerator_desc,
+         settings.benchmark_setting().accelerator_desc().c_str());
+  c_settings.accelerator_desc = accelerator_desc;
```

Comment:
I see that this piece of code is compliant and the "compliant solution" section suggests the same solution as what has been implemented. 

Also, this piece of code is same as what is being done for "accelerator" couple of lines above. Not sure how to get this fixed.

Reply:
> this piece of code is same as what is being done for "accelerator" couple of lines above.

The scanner only raises warning for new code.

> I see that this piece of code is compliant and the "compliant solution" section suggests the same solution as what has been implemented.

I think the code scanner is not smart enough, it just sees that there is a use of `strcpy` and raise a warning. Anyway, I think the alternative `strlcpy` is more safe to work with and would resolve the warning. But I can do that later in another PR since there are some more places to change.